content/en/metrics/metrics-without-limits.md

```diff
@@ -52,18 +54,52 @@ When configuring tags for counts, rates, and gauges, the most frequently queried
 
 **Note:** For tags to be managed on a metric, the metric must have a type declared. This is typically done when a metric is submitted, but may also be done manually using the `Edit` button for a metric in Metrics Summary.
 
+#### Use the API
+
+You can [create][2], [edit][3], [delete][4], and [estimate the impact][5] of your tag configuration through the Metrics APIs.
+
 ### Configure multiple metrics at a time
 
-Optimize your custom metrics volumes by using the [bulk metric tag configuration feature][7]. To specify a namespace for your metrics, click **Configure Tags*** on Metrics Summary. You can configure all metrics matching that namespace prefix with the same allowlist of tags under ***Include tags...*** or the same blocklist of tags under ***Exclude tags...***.
+Optimize your custom metrics volumes by using the [bulk metric tag configuration feature][7]. To specify the metrics to be configured, click **Configure Metrics**, then **Manage Tags*** on the Metrics Summary page. Select the metric or metric namespace you want to configure, then choose to do either of the following:
+   - [Allow all tags](#allow-all-tags) to override any previous tag configurations and make all tags queryable.
+   - [Include or exclude tags](#include-or-exclude-tags) to define queryable and non-queryable tags, respectively.
+
+#### Allow all tags
+
+{{< img src="metrics/bulk_allow_all_tags.png" alt="The Manage Tags option with Allow all tags selected in the Configure tags section" style="width:100%" >}}
+
+This option is selected by default, and overrides previously set tag configurations to make all tags queryable.
+
+#### Include or exclude tags
+
+When selecting tags to include or exclude, choose to either [override existing tag configurations](#override-existing-tag-configurations) or [keep existing tag configurations](#keep-existing-tag-configurations).
+
+##### Override existing tag configurations
+
+{{< img src="metrics/bulk_include_tags.png" alt="The Manage Tags option with Include tags and Override selected in the Configure tags section. The options to include tags actively queried on dashboards and monitors from the past 90 days and Specific tags are selected" style="width:100%" >}}
+
+All existing tag configurations for the selected metrics are overridden, and you define a new tag configuration. This allows you to make all tags queryable on all metric names. If you are choosing to **include tags**, you can select to include either or both of the following:
+   - Tags actively queried in Datadog from the past 30, 60, or 90 days.
+   - A specific set of tags that you define.
+
+##### Keep existing tag configurations
+
+{{< img src="metrics/bulk_exclude_tags.png" alt="The Manage Tags option with Exclude tags and Keep selected in the Configure tags section" style="width:100%" >}}
+
+Existing tag configurations are retained, and you define new tags to be added to the configuration.
+
+#### Use the API
```

Comment:
Again, this isn't a showstopper, but I typically try to keep all of my headers unique for accessibility reasons (if a user is scrolling through header links using a screenreader, having unique link text helps them from getting lost). This isn't always doable, and I totally see why you have multiple with this text in this topic, but it could be worth some thought about whether you can make them unique down the road.